src/apps/reservation-list/reservation-material/reservation-material.tsx

```diff
@@ -41,30 +41,23 @@ const ReservationMaterial: FC<ReservationMaterialProps & MaterialProps> = ({
     branches = excludeBlacklistedBranches(inputBranches, blacklistBranches);
   }
 
-  const openDetailsModal = useCallback(
-    (e: MouseEvent) => {
-      e.stopPropagation();
-      open(faust || identifier || "");
-    },
-    [faust, identifier, open]
-  );
+  const openDetailsModal = useCallback(() => {
+    open(faust || identifier || "");
```

Comment:
I guess it makes no sense to `open("")`.
how about:
```typescript
if (faust || identifier) {
   open(faust || identifier);
}

Reply:
sure